package.json

```diff
@@ -17,37 +18,56 @@
     "@types/react-dom": "17.0.2"
   },
   "dependencies": {
-    "@ethereumjs/block": "^5.1.1",
-    "@ethereumjs/common": "^4.2.0",
-    "@ethereumjs/evm": "^2.2.1",
+    "@babel/core": "^7.0.0",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
+    "@ethereumjs/block": "^5.2.0",
+    "@ethereumjs/common": "^4.3.0",
+    "@ethereumjs/evm": "^3.0.0",
     "@ethereumjs/tx": "^5.2.1",
     "@ethereumjs/util": "^9.0.2",
-    "@ethereumjs/vm": "^7.2.1",
+    "@ethereumjs/vm": "^8.0.0",
     "@kunigi/string-compression": "1.0.2",
-    "assert": "^2.1.0",
-    "buffer": "^6.0.3",
-    "classnames": "^2.5.1",
-    "copy-to-clipboard": "^3.3.3",
+    "@mdx-js/loader": "^2.3.0",
+    "@mdx-js/react": "^2.3.0",
+    "@monaco-editor/react": "^4.6.0",
+    "@mui/icons-material": "^5.15.2",
+    "@mui/material": "^5.15.2",
+    "@mui/system": "^5.8.0",
+    "@mui/x-tree-view": "^6.17.0",
+    "@solidity-parser/parser": "^0.18.0",
+    "assert": "^2.0.0",
+    "babel-preset-es2020": "^1.0.2",
+    "babelify": "^10.0",
+    "classnames": "^2.3.1",
+    "clsx": "^2.0.0",
+    "copy-to-clipboard": "^3.3.1",
     "ethereumjs-abi": "^0.6.8",
     "ethereumjs-util": "^7.1.5",
     "events": "^3.3.0",
     "gray-matter": "^4.0.3",
     "highlight.js": "^11.9.0",
     "highlightjs-solidity": "^2.0.6",
-    "kbar": "^0.1.0-beta.16",
+    "kbar": "^0.1.0-beta.45",
     "lodash.debounce": "^4.0.8",
+    "monaco-editor": "^0.47",
```

Comment:
Doesnt matter much but should represent proper semver structure instead of only `x.y`.